content/pages/status/index.md

```diff
@@ -24,12 +24,13 @@ Last Update : May 6, 2021 17:28 UTC
 | [mvis](/posts/mvis)  | ✅  | Everything works | 2021.504.1 (May 4, 2021) | [⬇️](https://github.com/EVAST9919/lazer-m-vis/releases/latest) |
 | [hitokori](/posts/hitokori)  | ✅ | Everything works | 0.18.5 (April 26, 2021) | [⬇️](https://github.com/Flutterish/Hitokori/releases/latest) |
 | [bosu](/posts/bosu)  | ✅ | Everything works | 2021.502.0 (May 2, 2021) | [⬇️](https://github.com/EVAST9919/bosu/releases/latest) |
-| [cytosu](/posts/cytosu)  | ⚠️ | Autoplay mod causes client to crash. | 2021.417.0 (April 17, 2021)| [⬇️](https://github.com/GSculerlor/Cytosu/releases/latest) |
+| [cytosu](/posts/cytosu)  | ✅ | Everything works | 2021.417.0 (April 17, 2021)| [⬇️](https://github.com/GSculerlor/Cytosu/releases/latest) |
```

Comment:
The status changed but version didn't. Was it a mistake previously?

Reply:
I test a ruleset on last version of osu!lazer and it's like in status that I write, then I try to install it again and it's work perfectly. It's my fault not check it correctly about rulesets version.